src/flamenco/genesis/fd_genesis_cluster.h

```diff
@@ -0,0 +1,33 @@
+#ifndef HEADER_fd_src_flamenco_genesis_fd_genesis_cluster_h
+#define HEADER_fd_src_flamenco_genesis_fd_genesis_cluster_h
+
+#include "../types/fd_types.h"
+
+#define FD_CLUSTER_UNKNOWN      (0UL)
+#define FD_CLUSTER_PYTHTEST     (1UL)
+#define FD_CLUSTER_TESTNET      (2UL)
+#define FD_CLUSTER_DEVNET       (3UL)
+#define FD_CLUSTER_PYTHNET      (4UL)
+#define FD_CLUSTER_MAINNET_BETA (5UL)
+
+/* Convert a base58 encoded hash to a FD_CLUSTER_* macro.
+
+   genesis_hash should point to a non-NULL cstr. It expects a
```

Comment:
Extreme style nit but these code comments should be double spaced, eg

```suggestion
   genesis_hash should point to a non-NULL cstr.  It expects a
```